packages/cli/package.json

```diff
@@ -46,6 +47,7 @@
     "semver": "^5.5.0",
     "truffle-config": "^1.0.6",
     "truffle-resolver": "^4.0.4",
+    "web3": "^1.0.0-beta.37",
```

Comment:
The Web3 dependency is used in cli only to map addresses to checksummed addresses in tests, so after we migrate to Truffle 5, it could be removed altogether.